Makefile

```diff
@@ -210,20 +210,11 @@ jsonnet: ## Download jsonnet locally if necessary.
 GINKGO = $(shell pwd)/bin/ginkgo
 ginkgo: ## Download ginkgo and gomega locally if necessary.
 	$(call go-get-tool,$(GINKGO),github.com/onsi/ginkgo/v2/ginkgo)
-	$(call go-get-tool,$(GOMEGA),github.com/onsi/gomega/...)
```

Comment:
- is removal of GOMEGA installation intentional?
- is it due to GINKGO also installing GOMEGA?

Reply:
>is removal of GOMEGA installation intentional?
yes it is, because we don't require a binary of GOMEGA and we already have it in our go.mod